content/sources/workflow.adoc

```diff
@@ -40,18 +42,39 @@ Branching model are 6 kinds of branch :
 
 === Develop Branch
 
-Develop branch is the production ready branch which is, in general case, the CI/CD production branch.
+For meeds-io repositories, develop branch is the branch containing all finished and tested features.
+For exoplatform repositories, develop branch is the production-ready branch which is, in general case, the CI/CD production branch.
 
 It contains the latest delivered development changes. This is our backbone where all the different fix and new feature are mixed with each other.
 
+For meeds-io repositories, when members finish a feature, they merge it on develop branch from their develop-xxx branch. Then, on a regular basis, a rebase is done to propagate these new changes on other members' develop branch :
+
 [[git-workflow-develop-branch]]
-.Git Workflow - Develop Branch
-image::git-workflow-develop-branch.png[Git Workflow - Develop Branch, 600, align=center, link={imagesdir}/git-workflow-develop-branch.png]
+.Git Workflow - Rebase Develop Branch
+image::git-worflow-rebase-develop.png[Git Workflow - Rebase Develop Branch, 800, align=center, link={imagesdir}/git-worflow-rebase-develop.png]
 
 === Feature Branch
 
 Feature branches are dedicated branch to develop a new feature.
 
+On meeds-io repositories, feature branches are created from develop-xxx branch (develop-exo for and eXo Platform feature).
+On exoplatform repositories, feature branches are created from develop branch.
+
 [[git-workflow-feature-branch]]
+.Git Workflow - Meeds Feature Branch
+image::git-workflow-meeds-feature-branch.png[Git Workflow - Meeds Feature Branch, 800, align=center, link={imagesdir}/git-workflow-meeds-feature-branch.png]
+
+In meeds repositories, when a User Story (US) is finished, it is backported to develop-xxx branch. So that, the nightly deployment will push this US in production for the association member which develop it, without impacting other association members.
```

Comment:
```suggestion
In meeds repositories, when a User Story (US) is completed, it is backported to develop-xxx branch. So that, the nightly deployment will attempt to push the feature in production for the association member which develops it, without impacting other association members.
```